internal/cmd/base/labels.go

```diff
@@ -20,8 +20,10 @@ type LabelCmds struct {
 	ShortDescriptionRemove string
 	NameSuggestions        func(client hcapi2.Client) func() []string
 	LabelKeySuggestions    func(client hcapi2.Client) func(idOrName string) []string
-	FetchLabels            func(s state.State, idOrName string) (map[string]string, int64, error)
-	SetLabels              func(s state.State, id int64, labels map[string]string) error
+	Fetch                  func(s state.State, idOrName string) (any, error)
+	SetLabels              func(s state.State, resource any, labels map[string]string) error
+	GetLabels              func(resource any) map[string]string
+	GetIDOrName            func(resource any) string
```

Comment:
We could make `LabelCmds` generic over the resource structs to get rid of the `any` here